components/Common/PrevNextArrow/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/24/solid';
+import type { FC, SVGAttributes } from 'react';
+
+type PrevNextArrowProps = {
```

Comment:
In this case you want to make it an interface probably:


```ts
interface PrevNextArrowProps extends SVGAttributes<SVGElement> {
  type: 'previous' | 'next';
}
```

Reply:
This upsets lint as it's [configured](https://github.com/nodejs/nodejs.org/blob/main/.eslintrc.json#L56) to accept `type` rather than `interface` for all `.tsx` files.
```
4:11  error  Use a `type` instead of an `interface`  @typescript-eslint/consistent-type-definitions
```

I tried reading about this a bit:
https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases

> Almost all features of an interface are available in type, the key distinction is that a type cannot be re-opened to add new properties vs an interface which is always extendable.

I don't think we'll ever need to extend the props type. So, `type` might be more appropriate in this case, CMIIW.

Reply:
True, I forgot we enforce `type` usage.